test/common/main_interface_test.cc

```diff
@@ -13,6 +13,26 @@ typedef struct {
   absl::Notification on_exit;
 } engine_test_context;
 
+// This config is the minimal envoy mobile config that allows for running the engine.
+// There is nothing functional about the config, as the created stream is only used for
+// send_metadata.
```

Comment:
do you mind updating the comment, as this is now used in more places?

Reply:
`There is nothing functional about the config, as the created stream is only used for send_metadata.` This part of the comment (and similar parts in other deleted comments) could be left in the test itself explaining why the minimal config is being used.